css/style.css

```diff
@@ -0,0 +1,87 @@
+body {
+  background: #1c1a19;
+}
+
+a {
+  text-decoration: none;
+}
+
+li {
+  list-style: none;
+}
+
+.hamburger {
+  padding-right: 30px;
+}
+
+.bar {
+  display: block;
+  width: 25px;
+  height: 3px;
+  margin: 5px auto;
+  background-color: white;
+}
+
+.topnav {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  font-family: 'Roboto', sans-serif;
+  width: 100%;
+  position: fixed;
+  background: #3c3a39;
+}
+
+header a {
+  color: white;
+  padding: 10px;
+  text-decoration: none;
+  font-size: 17px;
+}
+
+.container {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  gap: 4px;
+}
+
+.main-content {
+  padding: 50px 20px 1px 15px;
+}
+
+.headline {
+  font-style: normal;
+  font-family: 'Crete Round', sans-serif;
```

Comment:
- [ ] I can see that you forgot to add the background image, So please add it as a `background-image` property to your headline section here in the CSS file you can know more about it [here](https://www.w3schools.com/cssref/pr_background-image.asp) and you can position it right following this [link](https://www.w3schools.com/cssref/pr_background-position.asp).

| Figma Design  | Your Design |
| ------------- | ------------- |
| ![image](https://user-images.githubusercontent.com/68796232/170720974-c4de610c-d02c-47ed-b60a-5cebafc462c5.png)  | ![image](https://user-images.githubusercontent.com/68796232/170721108-4c5f0b43-a022-4597-81ea-85472aec2bd7.png) |